src/object/cli_obj.c

```diff
@@ -1957,6 +2076,53 @@ obj_ec_recov_cb(tse_task_t *task, struct dc_object *obj,
 		D_ERROR("task %p "DF_OID" EC recovery failed "DF_RC"\n",
 			task, DP_OID(obj->cob_md.omd_id), DP_RC(rc));
 	}
+	return rc;
+}
+
+static int
+obj_ec_check_exist(tse_task_t *task, struct dc_object *obj, struct obj_auxi_args *obj_auxi)
```

Comment:
This seems similar as obj_retry_cb()?

Reply:
the difference it here it create new sub-task to do the check exist.

Reply:
This is why I prefer adding new RPC format, instead of using sub-tasks.